action.yml

```diff
@@ -25,6 +25,14 @@ inputs:
     description: "Build a tarball for easy ArXiV submission?"
     required: false
     default: true
+  build-diff-on-pull-request:
+    description: "Build the `latexdiff` version of a PDF on pull requests"
+    required: false
+    default: true
+  latexdiff-url:
+    description: "URL to download `latexdiff`"
+    required: false
+    default: "https://raw.githubusercontent.com/ftilmann/latexdiff/1.3.2/latexdiff"
```

Comment:
For longevity's sake, it might be wise to copy this Perl script into the `showyourwork` repo.

Reply:
Sounds good to me.